Spec/Auth.swift

```diff
@@ -884,24 +884,33 @@ class Auth : QuickSpec {
             }
             
             // RSA6
-            it("should allow all operations when capability is not specified") {
+            it("should omit capability field if it is not specified") {
                 let tokenParams = ARTTokenParams()
-                expect(tokenParams.capability) == "{\"*\":[\"*\"]}"
+                expect(tokenParams.capability).to(beNil())
```

Comment:
@SimonWoolf Default should be empty, right?

Reply:
yup, RSA6